tests/raftstore/test_bootstrap.rs

```diff
@@ -0,0 +1,35 @@
+// Copyright 2017 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use super::cluster::{Cluster, Simulator};
+use super::node::new_node_cluster;
+use super::util::*;
+
+fn test_bootstrap_idempotent<T: Simulator>(cluster: &mut Cluster<T>) {
+    // assume that there is a node  bootstrap the cluster and add region in pd successfully
+    cluster.add_first_region().unwrap();
+    // now  at same time start the another node, and will recive cluster is not bootstrap
+    // try to bootstrap with a new region
```

Comment:
I think we need to test both when pd is bootstrapped and pd is not bootstrapped.